tests/integration/test_integration_snapshots.py

```diff
@@ -152,13 +154,38 @@ def task2(tracer):
             p.join()
         tracer.shutdown()
 
-    with tracer.trace("parent"):
+    with tracer.trace("parent") as parent:
+        # need to manually sample before Popen
+        tracer._sampler.sample(parent)
         p = multiprocessing.Process(target=task, args=(tracer,))
         p.start()
         p.join()
     tracer.shutdown()
 
 
+@snapshot(async_mode=False)
+def test_tracer_trace_across_only_fork():
```

Comment:
This new test is flakey, sometimes the child span is missing, need to figure this out. Fork is actually instrumented to have sampler run before the fork, and seems to work properly.